### PR TITLE
interface: add basic keybinds

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
included defaults (all customisable):
- move forward/backward in history (ctrl+b, ctrl+f)
- cycle through items in sidebar (ctrl+n, ctrl+p)
- hide sidebar (ctrl+h)

### CODE REVIEW

File: pkg/interface/src/wdyr.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-if (process.env.NODE_ENV === 'development') {
+if (false && process.env.NODE_ENV === 'development') {
```
Comment (matildepark) [inline pkg/interface/src/wdyr.js:3]: Cruft

File: pkg/interface/src/logic/state/settings.ts [hunk 2]
```diff
@@ -60,7 +71,19 @@ const useSettingsState = createState<SettingsState>('Settings', {
   tutorial: {
     seen: true,
     joined: undefined
+  },
+  keyboard: {
```
Comment (jamesacklin) [inline pkg/interface/src/logic/state/settings.ts:75]: Suggestions:

History back: `ctrl + [`
History forward: `ctrl + ]`
Previous channel: `ctrl + ;`
Next channel: `ctrl + '`
Toggle sidebar: `ctrl + \`

Ergonomically:
image

Comment (urcades) [reply pkg/interface/src/logic/state/settings.ts:75]: I actually love these — @liam-fitzgerald whaddya think?

Comment (liam-fitzgerald) [reply pkg/interface/src/logic/state/settings.ts:75]: Sounds good although I haven't used a normal keyboard in a minute so not sure how those would feel in practice

Comment (urcades) [reply pkg/interface/src/logic/state/settings.ts:75]: What kind of keyboard ya using? 

I tested this on a thinkpad and macbook and a 70% mech keyboard, it's pretty cozy

Comment (jamesacklin) [reply pkg/interface/src/logic/state/settings.ts:75]: `~mirmus-dozryx/clack` is calling you @liam-fitzgerald

Comment (liam-fitzgerald) [reply pkg/interface/src/logic/state/settings.ts:75]: Kinesis Advantage

**Comment by urcades** [comment]
Unsure how trivial it is to extend this PR (or if extending it is welcome), but for record's sake, I've added a deeply desired keybinding set for notifications:

https://github.com/urbit/landscape/issues/871

**Review by matildepark — APPROVED** [review]
It's the basics. Feels nice, though.

**Pull request merged**